src/pages/Admin/Charity/EditProfile/index.tsx

```diff
@@ -57,11 +51,11 @@ function FormWithContext(props: TProfile & { id: number }) {
       publicUrl: props.image ?? "",
       preview: props.image ?? "",
     },
-    logo: { name: "", publicUrl: props.logo ?? "", preview: props.logo ?? "" },
+    logo: { name: "", publicUrl: props.logo, preview: props.logo },
```

Comment:
Same issue as mentioned on devops PR. We'll need to make sure that existing records with "" instead of null are handled properly.

Reply:
in this particular case, form values are okay to be initialized with `""` which `props.logo` type is 
```
//endowment from endowments_v2
{
  logo: string // or-empty
}
```

Reply:
I guess that's true. :thinking: